doc/articles/uno-development/building-uno-ui.md

```diff
@@ -15,6 +15,19 @@ This article explains how to build Uno.UI locally, for instance if you wish to c
 - Install (**Tools** / **Android** / **Android SDK manager**) all Android SDKs starting from 7.1 (or the Android versions `TargetFrameworks` [list used here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.BindingHelper.Android/Uno.UI.BindingHelper.Android.csproj))
 - Run [Uno.Check](https://github.com/unoplatform/uno.check) on your dev machine to setup .NET 6 Android/iOS workloads
 
+## Recommended Windows hardware
+Loading and building the Uno.UI solution is a resource intensive task. As a result, opening it in Visual Studio 2022 requires a minimum hardware configuration to avoid spending time waiting for builds.
+
+**Minimum configuration:**
+- Intel i7 (8th gen) or equivalent
+- 16 GB of RAM
+- 250GB of Fast SSD
+
+**Optimal configuration:**
+- Intel i9 or equivalent
+- 32 GB of RAN
+- 500GB M2 SSD
```

Comment:
I wonder how the size matters?

Reply:
I need to upgrade my rig. i7 8th Gen here, meaning I am running on the minimal configuration. 😓

Reply:
It tends to matter when multiple visual studio instances are loaded at the same time, the page file tends to grow very *very* large.